autonomi/src/client/high_level/data/private.rs

```diff
@@ -55,6 +56,45 @@ impl Client {
         Ok(data)
     }
 
+    /// Stream a blob of (private) data from the network. Returns an Iterator that yields chunks progressively.
+    /// Use this for large files to avoid loading everything into memory.
```

Comment:
this is slightly misleading, as the yield chunks if not got drained, will be held in memory.
For `large files`, it's still the `stream with flush to disk` being the best option and guarantee `NOT` loading all stuff into memory.

Suggest to `remove` this sentence, or maybe change to:
Use this for streaming service such as videa clip to avoid long-time loading.